2021/_layouts/default.html

```diff
@@ -27,8 +27,18 @@
     <meta property="og:title" content="{{site.data.homepage.og_title}}"/>
     <meta property="og:description" content="{{site.data.homepage.og_description}}"/>
     <meta property="og:type" content="website"/>
-    <meta property="og:image" content="{{ base }}{{site.data.homepage.og_image}}"/>
+    <meta property="og:image" content="{{ site.data.homepage.og_url }}{{site.data.homepage.og_image}}"/>
+    <meta property="og:url" content="{{ site.data.homepage.og_url }}" />
     <!-- end: Facebook Open Graph -->
+    <!-- start: Twitter cards -->
+    <meta name="twitter:card" content="summary_large_image">
+    <meta property="twitter:domain" content="{{ site.data.homepage.og_url }}">
+    <meta property="twitter:url" content="{{ site.data.homepage.og_url }}">
+    <meta name="twitter:site" content="@jbcnconf" />
```

Comment:
Could we create 2 variables (@jbcnconf and @barcelonajug) in our "config file" and use them here instead of hard-coding those values?

Reply:
sure, will do